<launch>

  <arg name="model" default="$(find lynx)/urdf/lynx.urdf"/>
  <arg name="gui" default="true" />
  <arg name="rvizconfig" default="$(find lynx)/rviz/urdf.rviz" />

  <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
  <param name="use_gui" value="$(arg gui)"/>

  <!-- <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" /> -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
  <node name="rad2deg_converter" pkg="lynx" type="ard_control.py" />
  <node name="js_control" pkg="lynx" type="js_control.py" />
  <node name="js_pub" pkg="lynx" type="js_pub.py" />
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />

</launch>
